res/schema.xml

```diff
@@ -504,4 +504,15 @@ METADATA
       ALTER TABLE library ADD COLUMN coverart_digest BLOB;
     </sql>
   </revision>
+  <revision version="34" min_compatbile="1">
+    <description>Add Macros</description>
+    <sql>
+      CREATE TABLE macros (
+        id INTEGER PRIMARY KEY AUTOINCREMENT,
+        track_id INTEGER NOT NULL REFERENCES library(id),
+        label TEXT DEFAULT '' NOT NULL,
+        state INTEGER DEFAULT 0 NOT NULL,
+        content BLOB NOT NULL);
```

Comment:
How to you migrate content? You might need a `content_version INTEGER` column.

Reply:
The content will always be a protobuffer, so that can be versioned if needed

Reply:
In fact not needed now. I didn't consider that we can add columns when needed later.